Automattic/jetpack
# Quiz Shortcode: Offer Ability to Display Quizzes Together

#### Changes proposed in this Pull Request:

* This would allow people to display quizzes as one instead of them being listed individually throughout the post by wrapping the shortcode in `[quiz-wrapper]`

<img width="706" alt="Screenshot 2021-08-06 at 17 17 38" src="https://user-images.githubusercontent.com/43215253/128543059-16a8f700-e718-4379-a072-19cd7baff691.png">

#### Jetpack product discussion
None, but I don't imagine this will be a particularly controversial change. In my view, it's a significant enhancement to the existing Quiz system, and it's something I've hoped would be implemented for quite a while. Currently, all the quizzes are listed individually, and although that option will remain, it can quickly clutter a post.

#### Does this pull request change what data or activity we track or use?


Nope.

#### Testing instructions:

With Shortcodes enabled, try this. You should be able to now click through the Quiz, as it should all be displayed together. One of the reasons I've wanted this is to try something like Who Wants to be a Millionaire on my own site - this would allow something more like that compared to the existing Quiz system.

```
[quiz-wrapper]

[quiz]
[question]£100: Complete the famous phrase - "the writing is on the..."
[/question]

[answer]Wall[/answer]
[wrong]Notebook[/wrong]
[wrong]Building[/wrong]
[wrong]Window[/wrong]
[/quiz]

[quiz]
[question]£200: On a computer, "ALT" is short for?
[/question]

[answer]Alternate[/answer]
[wrong]Altitude[/wrong]
[wrong]Altogether[/wrong]
[wrong]Alter[/wrong]
[/quiz]

[quiz]
[question]£300: When drawing straws, a person gets chosen by picking which straw?
[/question]

[answer]Shortest straw[/answer]
[wrong]Last straw[/wrong]
[wrong]Thickest straw[/wrong]
[wrong]Bent straw[/wrong]
[/quiz]

[quiz]
[question]£500: William Shakespeare included the line "To be or not to be - that is the question" in which play?
[/question]

[answer]Hamlet[/answer]
[wrong]The Merchant of Venice[/wrong]
[wrong]Macbeth[/wrong]
[wrong]Othello[/wrong]
[/quiz]

[quiz]
[question]£1000: In the Chronicles of Narnia series, what is the name of the lion who appears?
[/question]

[answer]Aslan[/answer]
[wrong]Balrus[/wrong]
[wrong]Dagen[/wrong]
[wrong]Igor[/wrong]
[/quiz]

[quiz]
[question]£2000: What is the name for common table sugar?
[/question]

[answer]Sucrose[/answer]
[wrong]Glucose[/wrong]
[wrong]Fructose[/wrong]
[wrong]Maltose[/wrong]
[/quiz]

[quiz]
[question]£4000: Cleopatra was said to have been killed by what?
[/question]

[answer]Snake[/answer]
[wrong]Strangulation[/wrong]
[wrong]Knife[/wrong]
[wrong]Tripping[/wrong]
[/quiz]

[quiz]
[question]£8000: How many stars are there on the Australian flag?
[/question]

[answer]6[/answer]
[wrong]7[/wrong]
[wrong]8[/wrong]
[wrong]9[/wrong]
[/quiz]

[quiz]
[question]£16,000: The Ghats mountains are located in what country?
[/question]

[answer]India[/answer]
[wrong]China[/wrong]
[wrong]Switzerland[/wrong]
[wrong]Italy[/wrong]
[/quiz]

[quiz]
[question]£32,000: The Mississippi River empties into the Gulf of Mexico directly from what state?
[/question]

[answer] Louisiana[/answer]
[wrong] Mississippi[/wrong]
[wrong]Texas[/wrong]
[wrong]Alabama[/wrong]
[/quiz]

[quiz]
[question]£64,000: What currency is used in Bahrain?
[/question]

[answer]Dinar[/answer]
[wrong]Riyal[/wrong]
[wrong]Dirham[/wrong]
[wrong]Rouble[/wrong]
[/quiz]

[quiz]
[question]£125,000: Which of these cetaceans is classified as a “toothed whale”?
[/question]

[answer]Sperm whale[/answer]
[wrong]Grey whale[/wrong]
[wrong]Humpback whale[/wrong]
[wrong]Minke whale[/wrong]
[/quiz]

[quiz]
[question]£250,000: In 1718, which pirate died in battle off the coast of what is now North Carolina?
[/question]

[answer]Blackbeard[/answer]
[wrong]Calico Jack[/wrong]
[wrong]Captain Kidd[/wrong]
[wrong]Bartholomew Roberts[/wrong]
[/quiz]

[quiz]
[question]£500,000: In mythology, which creature sprang from the blood of the Medusa?
[/question]

[answer]Pegasus[/answer]
[wrong]Centaur[/wrong]
[wrong]Hydra[/wrong]
[wrong]Penguin[/wrong]
[/quiz]

[quiz]
[question]£1,000,000: Who wrote the opera ‘The Thieving Magpie’?
[/question]

[answer]Rossini[/answer]
[wrong]Verdi[/wrong]
[wrong]Donizetti[/wrong]
[wrong]Puccini[/wrong]
[/quiz]

[/quiz-wrapper]
```
